### PR TITLE
feat(crux-ui): add node connection card

### PR DESCRIPTION
**Pull request by m8vago**
feat(curx-ui): add node connection card
fix(crux-ui): lint errors
fix(crux): node install script priviledge escalation

### CODE REVIEW

File: web/crux-ui/src/hooks/use-timer.ts [hunk 1]
```diff
@@ -12,7 +13,7 @@ const useTimer = (
 
   useEffect(() => {
     if (remaining > 0) {
-      timer.current = setTimeout(() => setRemaining(remaining - 1), 1000)
+      timer.current = setTimeout(() => setRemaining(remaining - 1), SECOND_IN_MILLIS)
```
Comment (polaroi8d) [inline web/crux-ui/src/hooks/use-timer.ts:16]: <3

**Review by polaroi8d — APPROVED** [review]
(informal) LGTM, I know you just formatted a file in `crux/` but please change the scope of the PR to `web` instead of `crux-ui`

**Review by balanceee — CHANGES_REQUESTED** [review]
Code looks good to me but i found some bugs while testing it. Will discuss you with in person

**Review by balanceee — APPROVED** [review]

File: web/crux-ui/src/hooks/use-timer.ts (hunk 1 above)
Comment (m8vago) [reply web/crux-ui/src/hooks/use-timer.ts:16]: ^^

**Pull request merged**